apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -775,16 +785,16 @@ const Ticket = new GQLListSchema('Ticket', {
             if (resolvedStatusId) {
                 calculateTicketOrder(resolvedData, resolvedStatusId)
 
-                const existedStatusId = get(existingItem, 'status', null)
+                const resolvedStatus = await TicketStatus.getOne(context, { id: resolvedStatusId })
```

Comment:
can we use here `getById`?
And below in `existedStatus`

Reply:
I will try